3963 dbt on astro (#3964)
astronomer/docs: astro/deploy-history.md

1. In the Astro UI, select a Deployment.
2. Click **Deploy History**

> For each deploy, the **Deploy History** table shows the user that made the deploy, when they made the deploy, what image they used, the type of deploy, and any descriptions they added to the deploy.

Reviewer: ```suggestion
For each deploy, the **Deploy History** table shows the user who made the deploy, when they made the deploy, what image they used, the type of deploy, and any descriptions they added to the deploy.
```

### Types of deploys

There are different types of deploys, which are shared in the Deploy History table. You can read more about the different deploy options in [Deploy code to Astro](deploy-code.md). These deploy types include:

    - Image only deploy

>     - DAG only deploy

Reviewer: ```suggestion
    - DAG-only deploy
```

1. In the Astro UI, select a Deployment.
2. Click **Deploy History**.
3. Locate the deploy you want to roll back to. In the **Rollback to** column for the deploy, click **Deploy**.
4. Provide a description for your rollback, then complete the confirmation to trigger the rollback.

> You can roll back to any deploy within the last three months. If your last deploy was more than three months ago, you can only roll back to that deploy. After the rollback completes, the **Deploy History** table shows your rollback deploy as the most recent deploy and includes both your rollback description and rollback deploy time. Your Docker image tag and DAG bundle names are the same as the previous version you rolled back to. The historic deploy that you rolled back to still appears in chronological order in the table.

Reviewer 2: ```suggestion
You can roll back to any deploy within the last three months. If your last deploy was more than three months ago, you can only roll back to that deploy. After the rollback completes, the **Deploy History** table shows your rollback deploy as the most recent deploy and includes both your rollback description and rollback deploy time. Your Docker image tag, DAG bundle name, and dbt bundles are the same as the previous version you rolled back to. The historic deploy that you rolled back to still appears in chronological order in the table.
```
